### PR TITLE
Add section on reporting security vulnerabilities

### PR DESCRIPTION
**Pull request by renner**
This patch is adding a section about reporting security vulnerabilities via Github private issues.

### CODE REVIEW

**Review by juliogonzalez — COMMENTED** [review]
LGTM, but I am wondering... Shouldn't we maybe add a bit more information  here? Such as "please do not discuss the vulnerability outside the report" or "After the fix is public we will be happy to provide you our public acknowledgment", or things like that?

**Comment by renner** [comment]
> LGTM, but I am wondering... Shouldn't we maybe add a bit more information here? Such as "please do not discuss the vulnerability outside the report" or "After the fix is public we will be happy to provide you our public acknowledgment", or things like that?

Okay, how about: "Report security vulnerabilities [here](https://github.com/uyuni-project/uyuni/security/advisories/new), we would be happy to give credits for your help with making Uyuni more secure." The first part (about discussing outside) should be common sense from my perspective.

File: pages/contact.html [hunk 1]
```diff
@@ -101,6 +101,8 @@ <h4><a href="#ml">Mailing lists</a></h4>
             <h4><a href="#bugs">Reporting bugs</a></h4>
             <p> Submit bugs to the issues tab located at our <a href="https://github.com/uyuni-project/uyuni/issues">Github repository</a>.</p>
             <div class="alert alert-warning" role="alert"> Verify that a duplicate of the issue does not already exist! </div>
+            <h4><a href="#vulnerabilities">Reporting security vulnerabilities</a></h4>
+            <p>Report security vulnerabilities <a href="https://github.com/uyuni-project/uyuni/security/advisories/new">here</a>, we would be happy to give you credits for your help with making Uyuni more secure.</p>
```
Comment (juliogonzalez) [inline pages/contact.html:105]:
```suggestion
            <p>Report security vulnerabilities by creating a <a href="https://github.com/uyuni-project/uyuni/security/advisories/new">security advisory</a>. We would be happy to give you credits for your help with making Uyuni more secure.</p>
```

**Review by juliogonzalez — COMMENTED** [review]
Small suggestion

**Review by juliogonzalez — APPROVED** [review]

**Comment by juliogonzalez** [comment]
LGTM. Merging.

**Pull request merged**